libbeat/processors/dissect/field.go

```diff
@@ -63,6 +93,9 @@ func (f baseField) Length() int {
 func (f baseField) Key() string {
 	return f.key
 }
+func (f baseField) DataType() string {
```

Comment:
Nit. Add a new line before.
```suggestion

func (f baseField) DataType() string {
```